third_party/2and3/flask/app.pyi

```diff
@@ -26,25 +27,25 @@ class Flask(_PackageBoundObject):
     app_ctx_globals_class: type = ...
     config_class: Type[Config] = ...
     testing: Any = ...
-    secret_key: Any = ...
+    secret_key: Union[Text, bytes, None] = ...
     session_cookie_name: Any = ...
-    permanent_session_lifetime: Any = ...
-    send_file_max_age_default: Any = ...
+    permanent_session_lifetime: timedelta = ...
+    send_file_max_age_default: timedelta = ...
     use_x_sendfile: Any = ...
     json_encoder: Any = ...
     json_decoder: Any = ...
-    jinja_options: Any = ...
+    jinja_options: Dict[str, Any] = ...
```

Comment:
I'd prefer to leave this as `Any` for now. The real solution is probably to make `ImmutableDict` in the werkzeug stubs generic and use that here.